docs/atomic/atomic.rst

```diff
@@ -40,6 +40,20 @@ the atomic package takes care to support all spectral units.
           199.0122     C IV   E1 ... 1/2-* 157000000.0 0.00 -   502481.80
 
 
+Troubleshooting
+===============
+
+If you are repeatedly getting failed queries, or bad/out-of-date results, try clearing your cache:
+
+.. code-block:: python
+
+    >>> from astroquery.atomic import AtomicLineList
+    >>> AtomicLineList.clear_cache()
+
+If this function is unavailable, upgrade your version of astroquery. 
+The `clear_cache` function was introduced in version 0.4.7.dev8479.
```

Comment:
use full API path, otherwise this causes all the sphinx failures we see. (fix this here and everywhere else below)